CMakeLists.txt

```diff
@@ -40,7 +40,7 @@ else ()
   add_compile_options(-Wa,-mbig-obj)
 endif ()
 
-add_library(YimMenu MODULE "${SRC_MAIN}")
+add_library(YimMenu MODULE "${SRC_MAIN}" "src/backend/looped/self/interaction_menu_freedom.cpp" "src/backend/looped/weapons/modify_explosion_radius.cpp")
```

Comment:
Why is this here?

Reply:
No idea tbh, don't know why that went through